docs/api/QUIC_STREAM_EVENT.md

```diff
@@ -125,6 +125,9 @@ Upon successful handling of this event, the event handler should return one of `
 
 An array of `QUIC_BUFFER`s containing received data.
 
+The lifetime of the `Buffers` array itself is limited to the scope of the callback: if the received
+data is handled asynchronously (`QUIC_STATUS_PENDING`), the `QUIC_BUFFER` array should be copied.
```

Comment:
Is it intentional the wording changed from "must" to "should"?

Reply:
Good catch, I changed to *must* after the fact and didn't fix this one.

```suggestion
data is handled asynchronously (`QUIC_STATUS_PENDING`), the `QUIC_BUFFER` array must be copied.
```